packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -49,7 +49,7 @@ export const IdentifierInput: React.FC<IdentifierInputProps> = ({ patientIdentif
       selectedSource: null,
       autoGeneration: false,
     } as PatientIdentifierValue);
-  }, [initialValue, setHideInputField]);
+  }, [fieldName, initialValue, patientIdentifier, setFieldValue]);
```

Comment:
Same here

Reply:
here, some dependency fields were missing. `setHideInputField ` is to be persistent.